Tensile/Tests/extended/bufferload_offset/rocblas_sgemm_bufferload_limit.yaml

```diff
@@ -1,3 +1,8 @@
+ # the problems sizes in this test is extremely
+ # skip some old cards, not sure if it's too large to be allocated
+TestParameters:
```

Comment:
Same here. If CI doesn't complain in the past then there's no need to skip test

Reply:
update